mongodb-advanced/config.js

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2015-2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const config = require('bedrock').config;
+
+// custom atabase configuration
+config.mongodb.name = 'mongodb_advanced'; // default: bedrock_dev
```

Comment:
This is the only configuration option that needs to be specified here for an example.  Also, prefix the name with `bedrock_examples_` here and in the other example.